man/docker-build.1.md

```diff
@@ -90,7 +90,7 @@ set as the **URL**, the repository is cloned locally and then sent as the contex
 **--memory-swap**=*MEMORY-SWAP*
   Total memory (memory + swap), '-1' to disable swap.
 
-**-c**, **--cpu-shares**=*0*
+**-c** (deprecated), **--cpu-shares**=*0*
```

Comment:
I think we should just remove them here; we have marked them deprecated on the deprecation page; doing it here again isn't needed.